introduction/site_survey.md

```diff
@@ -0,0 +1,353 @@
+   1. [`csi` Command Line Configuration Payload](#csi_command_line_configuration_payload)
+   1. [`csi` Configuration Payload Files](#csi_configuration_payload_files)
+   1. [`site-init` Customizations](#site-init_customizations)
+   1. [Application Nodes](#application_nodes)
+   1. [Filesystems](#filesystems)
+
+<a name="system_nodes_and_networks"></a>
+## 1. First Master Node (ncn-m001)
+
+The first master node (ncn-m001) is also called the PIT node early in the installation process, but later becomes ncn-m001.
+
+| Name | Value |
+| ---- | ----- |
+| Factory-installed Linux root password of ncn-m001 | |
+| Site-defined Linux root password of ncn-m001 | |
+| BMC or iLO username for ncn-m001 | |
+| BMC or iLO password for ncn-m001 | |
+| BMC or iLO IP address of ncn-m001 on site BMC network | |
+| BMC or iLO Default route/gateway for ncn-m001 on site BMC network | |
+| BMC or iLO Netmask for ncn-m001 on site BMC network | |
+| IP address for ncn-m001 primary Ethernet on site network | |
+| Default route/gateway for ncn-primary Ethernet m001 on site network | |
+| Netmask for ncn-m001 primary Ethernet on site network | |
+| Network Interface ncn-m001 primary Ethernet to become `lan0` | | 
+
+<a name="site_time_information"></a>
+## 2. Site Time information
+
+| Name | Value |
+| ---- | ----- |
+| Time zone | | 
+| First site NTP server | |
+| (Optional) Second Site NTP server | |
+| (Optional) Third Site NTP server | |
+
+<a name="site_DNS_information"></a>
+## 3. Site DNS Information
+
+| Name | Value |
+| ---- | ----- |
+| Domain name | |
+| System name | |
+| First site DNS server IP address | |
+| (Optional) Second site DNS server IP address | |
+| (Optional) Third site DNS server IP address | |
+
+   > Note: The name of the system becomes part of the subdomain which is used to access externally exposed services. For example, if the system is named testsystem, and the domain name is example.com, the subdomain would be testsystem.example.com. Site DNS would need to be configured to delegate requests for addresses in this domain to the DNS IP address on CAN for resolution.
+
+<a name="CAN_network_ranges"></a>
+## 4. CAN Network Ranges 
+
+Site must provide an address range for the Customer Access Network (CAN) and its subnets.
+
+| Name | Starting IP address | Netmask
+| ---- | ----- | --- |
+| CAN | | |
+| can-static-pool | | | 
+| can-dynamic-pool | | | 
+| DNS IP address on CAN| | | 
+| CAN gateway IP address | | | 
+
+   > Note: The DNS IP address on the CAN is the IP address used for the HPE Cray EX DNS service. Site DNS delegates the resolution for addresses in the HPE Cray EX Domain to this server. This IP address must be in the can-static-pool subnet.
+
+   > Note: The CAN gateway IP address is the IP address assigned to a specific port on the spine switch or edge switch, which will act as the gateway between the CAN and the rest of the customer's internal networks. This address would be the lasthop route to the CAN network.
```

Comment:
If **lasthop** isn't being used as a variable or command, then update it to "last hop".